contract-tests/tests/test/amazon/utils/application_signals_constants.py

```diff
@@ -12,6 +12,7 @@
 # Attribute names
 AWS_LOCAL_SERVICE: str = "aws.local.service"
 AWS_LOCAL_OPERATION: str = "aws.local.operation"
+AWS_REMOTE_DB_USER: str = "aws.remote.db.user"
```

Comment:
Contract tests do not seem to be updated to use this attribute.

Reply:
Will update